packages/__docs__/src/Heading/index.tsx

```diff
@@ -26,8 +26,9 @@ import React from 'react'
 import { Heading as UIHeading } from '@instructure/ui-heading'
 import { instructure } from '@instructure/ui-themes'
 import { EmotionThemeProvider } from '@instructure/emotion'
+import type { HeadingProps } from '@instructure/ui-heading/src/Heading/props'
```

Comment:
We should not use the full import path, always import from the top level of the package, e.g. in this case:
`import type { HeadingProps } from '@instructure/ui-heading'`. This is to prevent errors or big changes when we move packages around.